src/app/branded/step-1/branded-checkout-step-1.component.js

```diff
@@ -89,6 +93,9 @@ class BrandedCheckoutStep1Controller {
   submit () {
     this.resetSubmission()
     this.submitted = true
+    if (this.useV3 === 'true') {
+      this.$scope.$digest()
+    }
```

Comment:
When you bind events with `ng-*`, Angular knows about them and runs its dirty-checking logic after the event listener completes. Because this method was called in response to a React event that Angular doesn't know about, the change to `submitted` wasn't triggering the `$onChange` handlers. That's why we're manually forcing a recheck. I believe we could also do `$rootScope.$apply(() => { ... })` in `Recaptcha.tsx`, which is probably a better solution.

Reply:
Yes, using apply is a better way to do this, typically.